.travis.yml

```diff
@@ -15,7 +15,6 @@ install:
 - gem install sass
 - gem install jekyll -v 3.2.1
 script:
-- sbt ++$TRAVIS_SCALA_VERSION update
```

Comment:
When I modified this I saw it convenient to separate `update` into another process so as to measure how long it took, and also to try to minimize memory overflow failures in `sbt` in case the `update` left some garbage behind.
¿Is it better not to keep it separate?